CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Removed Pleblist StreamTip integration
```

Comment:
The streamtip integration is already defunct, you just removed some remnants. I think we shouldn't add a changelog entry for this reason